ShortestPath/ShortestPath/InterfaceExample.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
```

Comment:
This file was just an example for the interfaces topic, you can delete it

Reply:
I kept it to check it again. So I do remove it